core/src/test/java/kafka/test/junit/RaftClusterInvocationContext.java

```diff
@@ -189,7 +189,9 @@ public void start() {
                 if (started.compareAndSet(false, true)) {
                     clusterTestKit.startup();
                     kafka.utils.TestUtils.waitUntilTrue(
-                            () -> this.clusterTestKit.brokers().get(0).brokerState() == BrokerState.RUNNING,
```

Comment:
nice finding. Maybe you can file a MINOR patch for it, since this PR needs more time to be completed

Reply:
I open a new PR for this change.
Please take a look [here](https://github.com/apache/kafka/pull/16537)